doc/Tutorial-Minsky.tex

```diff
@@ -1,449 +1,418 @@
-employment response function by the current wage before we integrate
-it (see the last equation above). There are two ways to do
-this. First, place a multiply block between the wage change function
-and the integral block, wire the function up to one input on the
-multiply block, and then either:
-
+``{\em\textbf{w}'' for the wage rate. Either right click or double-click
+on ``int1'' and this will bring up the edit window . Rename it to
+``w'' and give it a value of 1:}
+\begin{center}
+\scalebox{0.5}{\htmladdimg{NewItem93.png}} 
+\par\end{center}
+
+To compete the integral equation, we need to multiply the linear employment
+response function by the current wage before we integrate it (see
+the last equation above). There are two ways to do this. First, place
+a multiply block between the wage change function and the integral
+block, wire the function up to one input on the multiply block, and
+then either:
 \begin{itemize}
-\item wire the output of the {\bf\em w} block back to the other input
-on multiply block; or
-\item Right-click on {\bf\em w}, choose ``Copy Var'', place that copy
-before the multiply block, and wire it up.
+\item wire the output of the \textbf{\em w} block back to the other input
+on multiply block; or 
+\item Right-click on \textbf{\em w}, choose ``Copy Var'', place that
+copy before the multiply block, and wire it up. 
 \end{itemize}
-
 The first method gives you this initial result:
-
 \begin{center}
-\includegraphics{images/NewItem110.eps}
-\end{center}
+\includegraphics{images/NewItem110} 
+\par\end{center}
 
 That looks messy, but notice the blue dot on the wire? Click and drag
-on that and you will turn the straight line connector into a curve: 
-
+on that and you will turn the straight line connector into a curve:
 \begin{center}
-\resizebox{\textwidth}{!}{\includegraphics{images/NewItem111.eps}}
-\end{center}
+\resizebox{\textwidth}{!}{\includegraphics{images/NewItem111}} 
+\par\end{center}
 
 The second approach, which I personally prefer (it's neater, and it
-precisely emulates the integral equation), yields this result: 
-
+precisely emulates the integral equation), yields this result:
 \begin{center}
-\resizebox{\textwidth}{!}{\includegraphics{images/NewItem112.eps}}
-\end{center}
-
-From this point on the model develops easily---``like money for old
-rope'', as one of my maths lecturers used to say. Firstly if we
-multiply the wage rate {\bf\em w} by {\bf\em Labor} we get the {\em\bf
-Wage Bill}. To do this,
-firstly create the variable Wage Bill, and put it well below where $w$
-currently is on your diagram: 
+\resizebox{\textwidth}{!}{\includegraphics{images/NewItem112}} 
+\par\end{center}
 
+From this point on the model develops easily---``like money for
+old rope'', as one of my maths lecturers used to say. Firstly if
+we multiply the wage rate \textbf{\em w} by \textbf{\em Labor} we
+get the {\em\textbf{Wage Bill}. To do this, firstly create the
```

Comment:
Missing close brace


Seems like Lyx copes with {\bf\em ...} but not {\em\bf ...}. They're both equivalent, and equivalent to \textbf{\emph{...}}